docs/export.md

```diff
@@ -0,0 +1,71 @@
+# Export
+
+ROBOT can export details about ontology entities as a table (TSV or CSV). At minimum, the `export` command expects an input ontology (`--input`), a set of columns (`--columns`), and a file to write to (`--export`):
+
+```
+robot export --input nucleus.owl \
+  --columns "CURIE LABEL" \
+  --export nucleus.csv
+```
+
+### Columns
+
+The `--columns` are space-separated special keywords or properties used in the ontology. These should always be enclosed in double quotes.
+
+* **Special Keywords**:
+	* `IRI`: creates an "IRI" column based on the full unique identifier
+	* `CURIE`: creates a "CURIE" column based on the short form of the unique identifier
+	* `LABEL`: creates a "Label" column based on `rdfs:label`
+	* `subclass-of`: creates a "SubClass Of" column based on `rdfs:subClassOf`
+	* `equivalent-classes`: creates an "Equivalent Classes" column based on `owl:equivalentClass`
+	* `subproperty-of`: creates a "SubProperty Of" column based on `rdfs:subPropertyOf`
+	* `equivalent-properties`: creates an "Equivalent Properties" column based on `owl:equivalentProperty`
+	* `disjoints`: creates a "Disjoint With" column based on `owl:disjointWith`
+	* `types`: creates an "Instance Of" column based on `rdf:type` for named individuals
+* **Property CURIES**: you can always reference a property by the short form of the unique identifier (e.g. `oboInOwl:hasDbXref`). ROBOT will attempt to find a label for a CURIE to set as the column header, but if the label is not defined in the ontology, the header will be the CURIE. Any prefix used [must be defined](global/prefixes).
+* **Property Labels**: as long as a property label is defined in the input ontology, you can reference a property by label enclosed in single quotes (e.g. `database_cross_reference`). This label will also be used as the column header.
+
+The first column in the `--columns` list is used to sort the rows of the export. You can change the column that is sorted on by including `--sort <property>`.
```

Comment:
We should allow multiple `--sort` options, and also reverse sort. These have to be handled in the order they are specified.

Reply:
Allow multiple sorts separated by the delimiter (comma, tab...). You can put a `*` in front of a header name to sort in reverse.